test/e2e/specs/editor/various/is-typing.spec.js

```diff
@@ -36,36 +36,4 @@ test.describe( 'isTyping', () => {
 		// Toolbar Popover is hidden again
 		await expect( blockToolbarPopover ).toBeHidden();
 	} );
-
-	test( 'should not close the dropdown when typing in it', async ( {
```

Comment:
@ellatrix do you think we should update this test to something like typing in the `link` dialog or something instead of removing it?